src/asr/asr_chainer.py

```diff
@@ -570,14 +513,20 @@ def recog(args):
         logging.info("prediction [%s]: " + seq_hat_text, name)
 
         # copy old json info
-        new_json[name] = recog_json[name]
+        new_json[name] = dict()
+        new_json[name]['utt2spk'] = recog_json[name]['utt2spk']
 
         # add 1-best recognition results to json
-        new_json[name]['rec_tokenid'] = " ".join(
+        out_dic = dict()
+        for _key in recog_json[name]['output'][0]:
+            out_dic[_key] = recog_json[name]['output'][0][_key]
+
+        out_dic['rec_tokenid'] = " ".join(
             [str(idx[0]) for idx in y_hat])
-        new_json[name]['rec_token'] = " ".join(seq_hat)
```

Comment:
If you mean that it is needed so then we can process the decoding,  then it is no longer necessary. I modified the json2trn.py so it can accept the new format.

Reply:
In previous version, there exists the 'rec_token' key.
In addition, in the current asr_pytorch.py, there also exists it.
My comment is mainly that it would be better to follow them.

Reply:
Sorry about this, I did not notice that I deleted. Thank you for the notification.